timescaledb/how-to-guides/replication-and-ha/configure-replication.md

```diff
@@ -0,0 +1,391 @@
+# Configure Replication
+This section outlines how to set up asynchronous streaming replication on one or
+more database replicas.
```

Comment:
@charislam Did you miss this file when you were reviewing? (I doubt my first draft is *that* good 😉 )

Reply:
Oh yes I did! Thank you!